01/01b_coin_flips.Rmd

```diff
@@ -69,8 +72,7 @@ parameters {
 }
 model {
     theta ~ beta(1, 1); // SPECIFY YOUR PRIOR HYPERPARAMETERS HERE
-    for (n in 1:N)
```

Comment:
Should we also vectorize this in the python version?

Reply:
We should yes. It should work the same for python but I don't have a working python environment with stan in it to check. I'll try to set one up tomorrow. Or if anyone gets to first they should feel free to change it.